lib/github.js

```diff
@@ -28,6 +28,10 @@ function put(url, body, options) {
     return _post("PUT", url, body, options);
 }
 
+function patch(url, body, options) {
```

Comment:
Fun fact, you could write this
```
patch = _post.bind(null, "PATCH");
```

Which unfortunately does also bind `this`, but that's harmless in this case.

Reply:
Thanks! I'll keep this as-is for consistency with other wrappers in the file.

(Thanks, also, for the speedy review!)